manifests/install.pp

```diff
@@ -35,6 +35,19 @@
       ensure  => present,
       require => Yumrepo['opendaylight-40-release'],
     }
+    ->
+    # Configure the systemd file to force ipv4 binds (instead of ipv6)
+    file_line { 'odl_start_ipv4 ':
+      ensure  => present,
```

Comment:
CI style checks are failing b/c of the spacing here. Need one space after longest var name, then other spacing to line up (currently have two spaces)